Formula/msc-generator.rb

```diff
@@ -0,0 +1,56 @@
+class MscGenerator < Formula
+  desc "Draws signalling charts from textual description"
+  homepage "https://sourceforge.net/p/msc-generator"
+  url "https://downloads.sourceforge.net/project/msc-generator/msc-generator/v6.x/msc-generator-6.3.9.tar.gz"
+  sha256 "87bcb87109dc7ddabbddd68ac361a12ad4f40d7787542d82411e85b61e53d814"
+  license "AGPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "bison" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "graphviz"
+
+  # A single change to move out inline definition of a templated member function -- GCC can do it
+  patch :p1, :DATA
+
+  def install
+    ENV["PATH"] = "#{ENV["HOMEBREW_OPT"]}/gcc/bin:#{ENV["PATH"]}"
+    system "./configure",
+             "CPPFLAGS=-I#{ENV["HOMEBREW_OPT"]}/glpk/include",
```

Comment:
You haven't added `glpk` as a build dependency. I don't think this would be needed if you did.